README.md

```diff
@@ -16,6 +16,13 @@ When workloads run on Kip, your cluster size naturally scales with the cluster w
 * [FAQ](#faq)
 * [How it Works](#how-it-works)
 
+## Requirements
+
+To build Kip you need:
+
+- golang 1.14+ (older versions may work)
+- deepcopy-gen: go install k8s.io/code-generator/cmd/deepcopy-gen
```

Comment:
To make sure `go.mod` is not updated locally: `(cd && GO111MODULE=on go get k8s.io/gengo/examples/deepcopy-gen)`

Reply:
Yep, I still need to learn more about this new Golang module tooling.

Back in 2017 I used Glide, it was hip and cool.

Happy to see Golang now has built-in tooling to do this. And to activate it one has to set an environment variable with an impossible to remember name x-D